make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
Well, as long as the comment is not added, the source maps aren't loaded by the browser? So no production impact, right?
I think there at least *was* an option to add a sourcemaps file to a javascript file in chrome devs tools, but I couldn't find it right now.